Enhanced_Crouch/client.lua

```diff
@@ -90,7 +90,7 @@ end
 
 RegisterCommand('crouch', function()
 	DisableControlAction(0, 36, true) -- magic
-	if not Cooldown then
+	if not Cooldown and not IsPedInAnyVehicle(PlayerPedId(), false) then
```

Comment:
it's not better to add this in ``CanCrouch`` function and call ``CanCrouch`` function in here? :D